Francis/main/morse.ino

```diff
@@ -0,0 +1,28 @@
+
+void a(){delay(2*unit);dot();dash();}
```

Comment:
I think you missed to put some comments on your code... I suggest you to put a loop with function to make your code shorter...